PYME/recipes/processing.py

```diff
@@ -1959,3 +1959,103 @@ def execute(self, namespace):
             pass
             
         namespace[self.output_name] = res
+
+@register_module('DarkAndVarianceMap')
+class DarkAndVarianceMap(ModuleBase):
+    input = Input('input')
+    output_variance = Output('variance')
+    output_dark = Output('dark')
+    dark_threshold = Float(10000)  # this really should depend on the gain mode (12bit vs 16 bit etc)
+    variance_threshold = Float(90000)  # again this is currently picked fairly arbitrarily
+    start = Int(0)
+    end = Int(0)
+    uniform = Bool(False)  # simulate a uniform map
```

Comment:
If we do go with a recipe module, it should definitely not support uniform maps (the use case for these is very limited - namely testing equivalence of the map and non-map code, and I think it's reasonable that the user should have to jump through hoops to generate them).